flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -1,9 +1,16 @@
 from collections import defaultdict
 import json
 from typing import Callable, Dict, List, Optional, Union
-from flaml import oai
+from flaml.autogen import oai
 from .agent import Agent
-from flaml.autogen.code_utils import DEFAULT_MODEL, UNKNOWN, execute_code, extract_code, infer_lang
+from flaml.autogen.code_utils import DEFAULT_MODEL, UNKNOWN, execute_code, extract_code, find_code, infer_lang
+
+try:
+    from termcolor import colored
+except ImportError:
+
+    def colored(x, *args, **kwargs):
```

Comment:
Maybe considering using a global logger for all `print` ?

Reply:
created an issue #1159